src/azure-cli-core/azure/cli/core/extension/__init__.py

```diff
@@ -184,6 +184,24 @@ def get_azext_metadata(ext_dir):
                 azext_metadata = json.load(f)
         return azext_metadata
 
+    @staticmethod
+    def get_metadata_extras(ext_file):
+        metadata_extras = []
+        extra_const = "; extra == "
```

Comment:
This matching pattern may not be accurate. For example, in `requests` library, its `setup.py` has 

https://github.com/psf/requests/blob/v2.27.1/setup.py#L111

```py
        'socks:sys_platform == "win32" and python_version == "2.7"': ['win_inet_pton'],
```

while the `requests-2.27.1.dist-info/METADATA` from the `whl` downloaded from https://pypi.org/project/requests/#files has

```
Requires-Dist: win-inet-pton ; (sys_platform == "win32" and python_version == "2.7") and extra == 'socks'
```

making `; extra ==` fail to match.

Reply:
good catch. Changing to match ` extra ==`